package.json

```diff
@@ -2,25 +2,46 @@
   "name": "single-spa",
   "version": "5.9.3",
   "description": "The router for easy microfrontends",
-  "main": "lib/umd/single-spa.min.js",
+  "main": "lib/umd/single-spa.min.cjs",
   "types": "typings/single-spa.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "development": {
+        "import": "./lib/esm/single-spa.dev.js",
```

Comment:
I'm using the `esm` directory rather than `es2015` to maintain IE11 support. One option could be to change it to `es2015` here and then ask users who support IE11 to use the `esm` version, as a step towards dropping IE11 support. Thoughts?

Reply:
What's the size difference between the two?

In any case, I think it may be worth dropping IE11 support by default - otherwise, we're stuck waiting for the next major to drop it, and who knows how long that will be?

Reply:
It's 2kb. I agree it would be nice to start indicating that we're not going to support IE11 forever by changing the default to not support IE11 but still have it supported with a bit of manual configuration.

Most people use the `system` version, though, which currently supports ie11. Maybe we could create a system2015 version, as well?

Reply:
Am leaning towards this directory structure:

```
lib/
  es5/
    umd/
    system/
    esm/
  es2015/
    umd/
    system/
    esm/
```

Reply:
That sounds good to me